Tools/MeshUpgrader/src/main.cpp

```diff
@@ -961,16 +950,20 @@ void resolveColourAmbiguities(Mesh* mesh)
             sm->vertexData->convertPackedColour(originalType, desiredType);
         }
     }
-
-
 }
 
 struct MaterialCreator : public MeshSerializerListener
 {
     void processMaterialName(Mesh *mesh, String *name)
     {
-        // create material because we do not load any .material files
-        MaterialManager::getSingleton().createOrRetrieve(*name, mesh->getGroup());
+		if(*name == "") {
```

Comment:
Why not `name->empty()`?

Reply:
No reason, if @paroj agrees I'll change it.

Reply:
generally, `string::empty()` is faster as `string::compare()` involves a call to `strlen`